about.html

```diff
@@ -0,0 +1,307 @@
+
+                            </p>
+                            <p><img src="./images/images/about_icon.png" alt="icon"></p>
+                            <p class="about_paragraph_box">April 13th - 14th, 2023 (Thu - Fri) <br>
+                                See you at the Catenx Summit!</p>
+
+                        </div>
+                    </div>
+                    <div class="row">
+                        <div class="col-12 ">
+                            <p class="about_buttom_paragraph">Other questions about West Africa Youth Agric-Conference? <br>
+                                <span class="organisation"><a href="#">okhaishiestevenix@yahoo.com</a> </span>
+                            </p>
+                        </div>
+                    </div>
+
+
+                </div>
+            </div>
+        </div>
+        <div>
+            <!-- ---------------------Middle section about page  -->
+            <div class="container-fluid about_middle_section">
+                <div class="row">
+                    <div class="col-4"></div>
+                    <div class="col-8">
+                        <div class="about_middle_container">
+                            <h2>Catenx Nigeria limited Logo</h2>
+                            <div class="row">
+                                <div class="col-2"></div>
+                                <div class="col-10">
+                                    <div class="about_divider"></div>
+                                </div>
+                            </div>
+                        </div>
+
+
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-2"></div>
+                    <div class="col-8 about_paragraph_container">
+                        <p>
+                            Through the logo contest,
+                            which was held from June 8th to July 7th, the winning design was determined through online
+                            voting and evaluation by judges.
+                            The logo of Catenx Nigeria Limited, which was optimized for use on the homepage and
+                            promotional materials after consultation with the winning designer, is as follows.
+                        </p>
+
+                    </div>
+                </div>
+                <div class="row border_line">
```

Comment:
- [ ] I noticed that this part is different from the original design. Please check the screenshots below :arrow_down:  to see the difference and make sure that you follow the original and give to your user a good experience :+1: 

#### Your design
![Screenshot_20230113_122759](https://user-images.githubusercontent.com/6615394/212320644-c755cf77-6b7f-46da-9723-c6d599e8c6ad.jpg)


#### Original design
![Screenshot_20230113_123017](https://user-images.githubusercontent.com/6615394/212321000-346f838a-1f68-4207-acaa-e3e8129c1b0c.jpg)